src/man/sssd.conf.5.xml

```diff
@@ -211,6 +216,13 @@
                                 <phrase condition="with_pac_responder">, pac</phrase>
                                 <phrase condition="with_ifp">, ifp</phrase>
                             </para>
+                            <para>
+                                <phrase condition="have_systemd">
+                                    By default, all services are disabled and the administrator
+                                    must enable the ones allowed to be used by executing:
+                                    "systemctl enable sssd-@service@.socket".
```

Comment:
This question is not directly related to these patches, but I wonder if we should also "productize" the patches in realmd or even ipa-client-install by not configuring the services at all but instead enabling the socket units that should otherwise be listed in the services line. (Of course not in RHEL)